data/private

```diff
@@ -126,34 +126,52 @@ full:instant.arubanetworks.com
 full:setmeup.arubanetworks.com
 
 # ASUS router
+# Reference: https://www.asus.com/support/FAQ/1005263/
+full:asusrouter.com
```

Comment:
Can not see the reason why we should introduce this domain into this category.

Reply:
https://user-images.githubusercontent.com/37755859/229323400-a2d20cd9-a686-4052-955a-26b3da6724a1.mp4

When ASUS started to use new domain `www.asusrouter.com` for admin page of the new firmware version of routers, the new without the `www` prefix and the legacy `router.asus.com` are also supported.

Now these two are aliases of the new domain, and inner service of the router will automatically redirect (not rewrite) to the new domain. 

![image](https://user-images.githubusercontent.com/37755859/229320509-f41ab6a8-08e7-448c-8973-298c0f926a3f.png)

`www.asusrouter.com` is resolved to `192.168.50.1`.
`asusrouter.com` and `router.asus.com` are rebinded to `10.0.0.1` and handled by inner service for further process.